kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js

```diff
@@ -92,22 +91,31 @@ define([
         let analysisType = '';
         const params = {};
         const outputParams = {};
+        // unfortunate data structure makes generating this data long-winded
         Object.keys(typesToFiles).forEach((type) => {
+            // match app_id to get the abbreviated name for the analysis
             if (typesToFiles[type].appId === jobInfo.app_id) {
                 analysisType = fileTypesDisplay.fileTypeMapping[type];
                 Object.keys(jobInfo.job_params[0]).forEach((param) => {
-                    const value = jobInfo.job_params[0][param];
-                    const label = appData.specs[jobInfo.app_id].parameters
-                        .filter((p) => {
-                            return param === p.id;
-                        })
-                        .map((p) => {
-                            return p.ui_name;
-                        })[0];
-                    params[param] = {
-                        value,
-                        label,
-                    };
+                    // find each parameter in job_params in the app specs
```

Comment:
The other part of this ticket: ensure that when the job params are displayed, only the user params are shown. We don't need to show any of the private/system variables.